tests/unit/services/external-links-test.js

```diff
@@ -19,12 +19,12 @@ module('Unit | Service | external-links', function (hooks) {
 
   test('billingUrl as organization', function (assert) {
     const service = this.owner.lookup('service:external-links');
-    assert.equal(service.billingUrl('organization', 'travis-ci'), 'https://billing.travis-ci.com/subscriptions/travis-ci');
+    assert.equal(service.billingUrl('organization', 'travis-ci'), 'https://travis-ci.com/account/subscription');
```

Comment:
This doesn't look right. For organization URL should be like https://travis-ci.com/organizations/travis-ci/subscription

Reply:
Thank you for info.
Yes, I also noticed this seeing the results of our tests.
It's corrected.